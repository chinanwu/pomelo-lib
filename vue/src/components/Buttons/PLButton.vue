<template>
  <PButton c-class="PLButton">
    <template #content>
      <slot name="content"></slot>
    </template>
  </PButton>
</template>

<script>
import PButton from "@/components/Buttons/PButton";

export default {
  name: "PLButton",
  components: {
    PButton
  }
};
</script>

<style lang="less">
.PLButton {
  font-size: 1.6rem;
}
</style>
