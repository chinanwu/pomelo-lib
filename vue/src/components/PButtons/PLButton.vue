<template>
  <PButton class="PLButton">
    <template #content>
      <slot name="content"></slot>
    </template>
  </PButton>
</template>

<script>
import PButton from "@/components/PButtons/PButton";

export default {
  name: "PLButton",
  components: {
    PButton
  }
};
</script>

<style lang="less" scoped>
.PLButton {
  font-size: 2rem;
}
</style>
