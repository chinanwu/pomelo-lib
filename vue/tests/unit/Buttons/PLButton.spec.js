import { shallowMount } from "@vue/test-utils";
import PLButton from "@/components/Buttons/PLButton";

describe("PLButton", () => {
  describe("rendering", () => {
    it("renders PLButton", () => {
      const wrapper = shallowMount(PLButton, {
        slots: {
          content: "Test"
        }
      });
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
