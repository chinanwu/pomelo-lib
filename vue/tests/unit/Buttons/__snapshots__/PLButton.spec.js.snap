// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLButton rendering renders PLButton 1`] = `<p-button-stub />`;
